app/presenters/contributions/index.rb

```diff
@@ -30,25 +33,23 @@ def stories_content
       }
     end
 
-    # TODO: i18n
     def stories_events
       @events.map do |event|
         {
           url: stories_path(event_id: event.id),
```

Comment:
`contributions_path`
otherwise you won't be able to switch events in the listing